### PR TITLE
feat: add totalAssetValue in GetAccount API response

### PR DESCRIPTION
**Pull request by carlcarl**
### Description

Add totalAssetValue in GetAccount API response.

### Rationale

Account page need show the value

### Example

N/A

### Changes

N/A

/update-integration-keyfile

### CODE REVIEW

File: service/apiserver/internal/logic/account/getaccountlogic.go [hunk 1]
```diff
@@ -76,6 +77,9 @@ func (l *GetAccountLogic) GetAccount(req *types.ReqGetAccount) (resp *types.Acco
 		Nonce:  account.Nonce,
 		Assets: make([]*types.AccountAsset, 0, len(account.AssetInfo)),
 	}
+
+	totalAssetPrice := big.NewFloat(0)
```
Comment (forcodedancing) [inline service/apiserver/internal/logic/account/getaccountlogic.go:81]: `price` seems to be not a good name. How about `totalAssetAmount` or `totalAssetValue`? Can you refer to some dapp (e.g., pancake, uniswap) for the name

File: service/apiserver/internal/logic/account/getaccountlogic.go [hunk 2]
```diff
@@ -104,9 +108,23 @@ func (l *GetAccountLogic) GetAccount(req *types.ReqGetAccount) (resp *types.Acco
 				Balance: asset.Balance.String(),
 				Price:   strconv.FormatFloat(assetPrice, 'E', -1, 64),
 			})
+
+			// BNB for example:
+			//   1. Convert unit of balance from wei to BNB
+			//   2. Calculate the result of (BNB balance * price per BNB)
+			balanceInFloat := new(big.Float).SetInt(asset.Balance)
+			unitConversion := new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
```
Comment (forcodedancing) [inline service/apiserver/internal/logic/account/getaccountlogic.go:116]: Different assets have different units. `Decimal` field of asset is for this, i believe.

File: docs/api_reference.md [hunk 3]
```diff
@@ -449,14 +449,15 @@ Send raw transaction
...
-| assets | [ [AccountAsset](#accountasset) ] |  | Yes |
+| Name              | Type                              | Description | Required |
+|-------------------|-----------------------------------| ----------- | -------- |
+| status            | integer                           |  | Yes |
+| index             | long                              |  | Yes |
+| name              | string                            |  | Yes |
+| pk                | string                            |  | Yes |
+| nonce             | long                              |  | Yes |
+| assets            | [ [AccountAsset](#accountasset) ] |  | Yes |
+| total_asset_price | string                            |  | Yes |
```
Comment (forcodedancing) [inline docs/api_reference.md:460]: Need to change the doc

**Review by forcodedancing — CHANGES_REQUESTED** [review]

File: service/apiserver/internal/logic/account/getaccountlogic.go (hunk 1 above)
Comment (guagualvcha) [reply service/apiserver/internal/logic/account/getaccountlogic.go]: `totalAssetValue ` seems better

Comment (carlcarl) [reply service/apiserver/internal/logic/account/getaccountlogic.go]: Oh, OK.
Thanks 👍

**Review by guagualvcha — APPROVED** [review]

**Review by forcodedancing — APPROVED** [review]

**Pull request merged**